lms/envs/bok_choy_static.py

```diff
@@ -0,0 +1,7 @@
+"""
+Settings for building static assets for Bok Choy tests.
+"""
+
+# TODO: update the LMS pipeline for Bok Choy to use compiled assets
```

Comment:
I'll make this comment clearer. For the moment I have left LMS with the old behavior (using the sources from the source tree instead of from the static root). When we implement RequireJS and Optimizer in LMS (next up in this sprint) we'll implement the support like we did for Studio. I had to add this file, though, because I changed all Bok Choy runs to expect a file of this name.